2014/12/03 05:14:26 [error] [exception.CHttpException.404] exception 'CHttpException' in C:\newOpenServer\domains\ybugs.local\protected\controllers\IssuesController.php:107
Stack trace:
#0 [internal function]: IssuesController->actionAdd('')
#1 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(IssuesController), Array)
#2 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(IssuesController), Object(ReflectionMethod), Array)
#3 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('add')
#7 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('issues/add/id')
#8 C:\newOpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\newOpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/issues/add/id/
---
2014/12/03 05:14:32 [error] [exception.CHttpException.404] exception 'CHttpException' in C:\newOpenServer\domains\ybugs.local\protected\controllers\IssuesController.php:107
Stack trace:
#0 [internal function]: IssuesController->actionAdd(NULL)
#1 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(IssuesController), Array)
#2 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(IssuesController), Object(ReflectionMethod), Array)
#3 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('add')
#7 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('issues/add')
#8 C:\newOpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\newOpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/issues/add/
---
2014/12/03 22:08:50 [error] [exception.CException] exception 'CException' with message 'ExtUserRoles and its behaviors do not have a method or closure named "getUseRole".' in C:\newOpenServer\domains\ybugs.local\framework\base\CComponent.php:266
Stack trace:
#0 C:\newOpenServer\domains\ybugs.local\framework\db\ar\CActiveRecord.php(227): CComponent->__call('getUseRole', Array)
#1 C:\newOpenServer\domains\ybugs.local\protected\controllers\UsersController.php(110): CActiveRecord->__call('getUseRole', Array)
#2 C:\newOpenServer\domains\ybugs.local\protected\controllers\UsersController.php(110): ExtUserRoles->getUseRole('1')
#3 [internal function]: UsersController->actionAddPrj('1')
#4 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#5 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#6 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('addprj')
#10 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('users/addprj')
#11 C:\newOpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 C:\newOpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/en/users/addprj/1
HTTP_REFERER=http://ybugs.local/en/users/list
---
2014/12/03 22:13:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/list".' in C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('user/list')
#1 C:\newOpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\newOpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/en/user/list
HTTP_REFERER=http://ybugs.local/en/users/addprj/1
---
2014/12/03 21:20:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "project/del".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('project/del')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/en/project/del
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/03 21:20:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "project/del".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('project/del')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/en/project/del
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/03 21:21:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "project/del".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('project/del')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/en/project/del
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/03 21:21:46 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('del')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('projects/del')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/en/projects/del
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/03 21:42:41 [error] [php] strpos() expects parameter 1 to be string, object given (/Applications/AMPPS/www/ybugs.local/framework/web/CHttpRequest.php:801)
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/protected/controllers/ProjectsController.php(99): ProjectsController->redirect()
#1 unknown(0): ProjectsController->actionDel()
#2 /Applications/AMPPS/www/ybugs.local/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs()
#3 /Applications/AMPPS/www/ybugs.local/framework/web/actions/CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(308): CInlineAction->runWithParams()
#5 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): ProjectsController->runAction()
#6 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): ProjectsController->runActionWithFilters()
#7 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): ProjectsController->run()
#8 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController()
#9 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /Applications/AMPPS/www/ybugs.local/index.php(13): CWebApplication->run()
REQUEST_URI=/en/projects/del/10
in /Applications/AMPPS/www/ybugs.local/protected/controllers/ProjectsController.php (99)
in /Applications/AMPPS/www/ybugs.local/index.php (13)
2014/12/03 21:43:14 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('del')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('projects/del')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/en/projects/del/list
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/04 01:38:06 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('delres')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/delres/1...')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/issues/delres/15
HTTP_REFERER=http://ybugs.local/en/issues/getissue/23
---
2014/12/04 01:38:08 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('delres')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/delres/1...')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/issues/delres/15
HTTP_REFERER=http://ybugs.local/en/issues/getissue/23
---
2014/12/04 01:38:08 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('delres')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/delres/1...')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/issues/delres/15
HTTP_REFERER=http://ybugs.local/en/issues/getissue/23
---
2014/12/04 01:38:08 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('delres')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/delres/1...')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/issues/delres/15
HTTP_REFERER=http://ybugs.local/en/issues/getissue/23
---
